grunt/tasks/jasmine.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
     return {
 
       'src': {
-        src: [], // actual src files are require'd in the *.spec.js files
+        src: [
+          'dist/cartodb.uncompressed.js',
+        ],
```

Comment:
Basically, some tests under `src` use `require('torque')` which then requires some files (`cdb./geo/gmaps/gmaps-torque-layer-view` and ``cdb/geo/leaflet/leaflet-torque-layer`) that rely on `cdb._`, which is defined by `cartodb.js`... Thoughts?